CMakeLists.txt

```diff
@@ -1,27 +1,40 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(pcl_conversions)
 
-find_package(catkin REQUIRED COMPONENTS pcl_msgs roscpp sensor_msgs std_msgs cmake_modules)
+if(NOT WIN32)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra")
+endif()
+
+find_package(ament_cmake REQUIRED)
+find_package(ament_cmake_gtest REQUIRED)
 
+find_package(Boost REQUIRED)
+find_package(builtin_interfaces REQUIRED)
+find_package(Eigen3 REQUIRED)
+# TODO(clalancette): Re-enable this once we have pcl_msgs
+#find_package(pcl_msgs REQUIRED)
 find_package(PCL REQUIRED QUIET COMPONENTS common)
```

Comment:
Heh, good question, that is weird.  Reading: https://cmake.org/pipermail/cmake/2007-June/014834.html, it seems like those keywords set an XXX_FIND_REQUIRED and XXX_FIND_QUIET variable before invoking the make.  So I guess it just depends on which order the "if" statements are in during the code.  @wjwwood You actually put this in initially; what's the semantics and/or reasoning behind this?